scriptmodules/ports/ut.sh

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env bash
+
+# This file is part of RetroPie-Extra, a supplement to RetroPie.
+# For more information, please visit:
+#
+# https://github.com/RetroPie/RetroPie-Setup
+# https://github.com/Exarkuniv/RetroPie-Extra
+#
+# See the LICENSE file distributed with this source and at
+# https://raw.githubusercontent.com/Exarkuniv/RetroPie-Extra/master/LICENSE
+#
+# This game works on i386, arm64, and amd64 package architectures
+#
+rp_module_id="ut"
+rp_module_desc="Unreal Tournament"
+rp_module_licence="PROP https://github.com/OldUnreal/UnrealTournamentPatches/blob/master/LICENSE.md"
+rp_module_help="Install game data by coping the contents of Maps/ Music/ Sounds/ Textures/ into their respective directories in $romdir/ports/ut/
+
+Make sure you do not overwrite the Texture files that came with the patch, otherwise the in-game text will render incorrectly.
+
+Post installation:
+Open '$home/.utpg/UnrealTournament.ini' and go to the section:
+
+[SDLDrv.SDLClient]
+WindowedViewportX=1024
+WindowedViewportY=768
+WindowedColorBits=32
+FullscreenViewportX=1920
+FullscreenViewportY=1080
+FullscreenColorBits=32
+
+Ensure the Fullscreen* Variables match your resolution.
+
+Note: Some display problems may be solved by removing '$md_inst/System64/libSDL2-2.0.so.0' and creating a symlink pointing to the system libSDL2-2.0.so.0, eg '/usr/lib/$(uname -m)-linux-gnu/libSDL2-2.0.so.0'  This is not a guarantee, but if you have trouble, start with this.
+"
+rp_module_section="exp"
+rp_module_repo="https://github.com/OldUnreal/UnrealTournamentPatches/"
+rp_module_flags="!all x86 64bit"
+
+function _get_branch_ut() {
+    local version=$(curl https://api.github.com/repos/OldUnreal/UnrealTournamentPatches/releases/latest 2>&1 | grep -m 1 tag_name | cut -d\" -f4 | cut -dv -f2)
+    echo -ne $version
+}
+
+function depends_ut() {
+    local depends="libsdl2-2.0-0 libopenal1"
+    getDepends "${depends[@]}"
```

Comment:
and since we are useing XINIT we need to add xorg to the depends

since this is only needed for the pi5 we should use this
``
	isPlatform "rpi5" && depends+=(xorg)
``

Reply:
Since this will, at least theoretically, run on an rpi3, I set this to ```isPlatform "rpi"```